openSUSE_install.md~

```diff
@@ -0,0 +1,218 @@
+![openSUSE Logo](https://github.com/openSUSE/artwork/blob/master/Logo/official-logo-color.png?style=centerme)
```

Comment:
That's some editor temporary file, I believe it should not be in the PR.

Reply:
Hi,
If an image link violates project policy or could somehow be made to be a security risk, I can certainly replace with text.
If an image link cannot be considered a security risk, I thought an image could enhance appearance.

Reply:
I meant `openSUSE_install.md~` which seems to be a temporary file for `openSUSE_install.md`. I believe only one of them (without `~`) has so be checked in.

Reply:
Thank you for your clarification, I removed all temporary files that were created by text editors.
Should be good to go now unless there is anything else.